### PR TITLE
Adding vpa exporter vpa to prevent OOMKills for vpa-exporter.

### PR DESCRIPTION
**Pull request by georgekuruvillak**
**What this PR does / why we need it**:
This PR enables `vpa` for `vpa-exporter` to prevent OOMKill of the vpa-exporter. `vpa-exporter` scraps `vpa` metrics from vpa resources in the shoots for Prometheus to injest.
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator
vpa added for vpa-exporter to prevent OOMKill of vpa-exporter pod.
```

### CODE REVIEW

File: charts/seed-bootstrap/charts/vpa/templates/vpa-exporter.yaml [hunk 1]
```diff
@@ -35,8 +35,8 @@ spec:
             cpu: 50m
             memory: 200Mi
           limits:
-            cpu: 200m
-            memory: 500Mi
+            cpu: 500m
```
Comment (wyb1) [inline charts/seed-bootstrap/charts/vpa/templates/vpa-exporter.yaml:39]: Can we remove the limits if vpa is enabled and keep them if it is disabled?

Comment (georgekuruvillak) [reply charts/seed-bootstrap/charts/vpa/templates/vpa-exporter.yaml:39]: Yes. I shall do that.

Comment (timuthy) [reply charts/seed-bootstrap/charts/vpa/templates/vpa-exporter.yaml:39]: Now the limits are removed entirely.
Like @wyb1 proposed, can you keep them if VPA is disabled?

Furthermore, what about the limits for `kube-state-metrics` @georgekuruvillak  and @wyb1?

Comment (wyb1) [reply charts/seed-bootstrap/charts/vpa/templates/vpa-exporter.yaml:39]: I propose simply putting an `if vpa not enabled` around the limits. 
```yaml
{{ if not .Values.vpa.enabled }}
limits:
    cpu: 200m
    memory: 500Mi
{{ end }}
```
We should do this for the vpa-exporter and kube-state-metrics.

Comment (georgekuruvillak) [reply charts/seed-bootstrap/charts/vpa/templates/vpa-exporter.yaml:39]: makes sense. I shall do that.

**Review by wyb1 — APPROVED** [review]
/lgtm

**Review by zanetworker — APPROVED** [review]

**Pull request merged**